gallery.html

```diff
@@ -7,9 +7,10 @@
 
 <div class="listing">
   {%- for discipline in site.galleries -%}
+    {%- assign discipline_lower = discipline.subheading | downcase | split: " " | first -%}
```

Comment:
Something to note: in some places (e.g. gallery, event config, folder names) `visual` is used, in other places `visual-arts`, hence the sketchiness of this line. Should probably use one or the other, which one?